x-pack/plugins/siem/public/common/components/exceptions/viewer/exception_details.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexItem, EuiFlexGroup, EuiDescriptionList, EuiButtonEmpty } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import { transparentize } from 'polished';
+
+import { ExceptionListItemSchema } from '../types';
+import { getDescriptionListContent } from '../helpers';
+import * as i18n from '../translations';
+
+const StyledExceptionDetails = styled(EuiFlexItem)`
+  ${({ theme }) => css`
+    background-color: ${transparentize(0.95, theme.eui.euiColorPrimary)};
+    padding: 16px;
+
+    .euiDescriptionList__title.listTitle--width {
+      width: 40%;
+    }
+
+    .euiDescriptionList__description.listDescription--width {
+      width: 60%;
+    }
+  `}
+`;
+
+const ExceptionDetailsComponent = ({
+  showComments,
+  onCommentsClick,
+  exceptionItem,
+}: {
+  showComments: boolean;
+  exceptionItem: ExceptionListItemSchema;
+  onCommentsClick: () => void;
+}) => {
```

Comment:
nit: I still recommend even though we don't have the linter rule to add return types to functions as a practice. Totally optional nit though.

Reply:
Good call. Updated on all 3 components returning `JSX.Element`.